src/historywork/test/HistoryWorkTests.cpp

```diff
@@ -38,7 +44,12 @@ TEST_CASE("write verified checkpoint hashes", "[historywork]")
         catchupSimulation.getClock().crank();
     }
 
-    Hash h = WriteVerifiedCheckpointHashesWork::loadHashFromJsonOutput(
-        pair.first, file);
-    REQUIRE(h == *pair.second);
+    for (auto const& p : pairs)
+    {
+        LOG(INFO) << "Verified " << p.first << " with hash "
```

Comment:
avoid default logging when running tests (this overwhelms Travis) : either use `DEBUG` level or use `INFO("Verified " << p.first << " with hash " << hexAbbrev(*p.second))` (as to give context in case of failure)

Reply:
Fixed.